app/services/idv/steps/send_link_step.rb

```diff
@@ -28,8 +28,22 @@ def call
         build_telephony_form_response(telephony_result)
       end
 
+      def extra_view_variables
+        {
+          idv_phone_form: create_form,
+        }
+      end
+
       private
 
+      def create_form
```

Comment:
I feel like we tend to call these constructor methods `build_` instead of `create_`
```suggestion
      def build_form
```

For ActiveRecord objects and FactoryBoy, `build` means "make it in memory" and `create` means "save it to the database" so since this is an in-memory only object I'd stick to build? Overall I don't feel that strongly.